.github/workflows/oblt-github-commands.yml

```diff
@@ -8,39 +8,21 @@
 name: oblt-github-commands
 
 on:
-  pull_request_target:
+  pull_request:
     types:
-      - opened
+      - labeled
 
 permissions:
   contents: read
+  pull-requests: write
 
 jobs:
-  comment-if-oblt-member:
+  comment:
+    if: ${{ github.event.label.name == 'ci:project-deploy-observability' }}
     runs-on: ubuntu-latest
     steps:
-      - uses: elastic/apm-pipeline-library/.github/actions/github-token@current
+      - uses: actions/github-script@v7
```

Comment:
Mind if we pin the sha similar to https://github.com/elastic/kibana/blob/main/.github/workflows/auto-approve-backports.yml#L18 and add it to [renovate.json](https://github.com/elastic/kibana/blob/02f277efa7470035671f67d4e86e74ea5f52838e/renovate.json#L40-L48)?

Reply:
I normally use `Pin actions to a tag only if you trust the creator` as mentioned in [here](https://docs.github.com/es/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions#using-third-party-actions) for `actions/github-script` or `actions/checkout` and so on. 

Shall I pin the sha even for a trusted action?

Reply:
Yeah understand, we're running with it for everything.  Long term it's moving to _all_ dependencies.

Reply:
See [`0060b46` (#195219)](https://github.com/elastic/kibana/pull/195219/commits/0060b462d0307929f6d63bc465690f01ea754cf7)